### PR TITLE
Fix - issue #948

### PR DESCRIPTION
**Pull request by BhargavBhandari90**
media_title attribute was not working for shortcodes

It was not showing title even it's value set to true

### CODE REVIEW

File: app/main/routers/query/RTMediaQuery.php [hunk 1]
```diff
@@ -676,13 +680,13 @@ function populate_media() {
...
 		}
 
-		// media title option
-		if ( isset( $this->media_query['media_title'] ) ) {
-			if ( 'false' === $this->media_query['media_title'] ) {
-				// Add filter show media title
+		// Media title option.
+		if ( isset( $this->media_query ) ) {
+			if ( ( isset( $this->media_query['media_title'] ) && 'false' === $this->media_query['media_title'] ) || ( true === $is_gallery_shortcode && ! isset( $this->media_query['media_title'] ) ) ) {
+				// Add filter show media title.
```
Comment (kiranpotphode) [inline app/main/routers/query/RTMediaQuery.php]: Just a suggestion you can add dock block for hooks as per this reference
https://github.com/buddypress/BuddyPress/blob/master/src/bp-core/bp-core-attachments.php#L105
@BhargavBhandari90

File: app/main/routers/query/RTMediaQuery.php [hunk 2]
```diff
@@ -676,13 +680,13 @@ function populate_media() {
 			unset( $this->media_query['lightbox'] );
 		}
 
-		// media title option
-		if ( isset( $this->media_query['media_title'] ) ) {
-			if ( 'false' === $this->media_query['media_title'] ) {
-				// Add filter show media title
+		// Media title option.
+		if ( isset( $this->media_query ) ) {
+			if ( ( isset( $this->media_query['media_title'] ) && 'false' === $this->media_query['media_title'] ) || ( true === $is_gallery_shortcode && ! isset( $this->media_query['media_title'] ) ) ) {
```
Comment (pradeep910) [inline app/main/routers/query/RTMediaQuery.php]: @BhargavBhandari90 `'false' === $this->media_query['media_title']`
I think `false` should be without single quotes. 
Please check again if its boolean or not.

Comment (BhargavBhandari90) [inline app/main/routers/query/RTMediaQuery.php]: This is a shortcode attribute value which will be a string.

Comment (pradeep910) [inline app/main/routers/query/RTMediaQuery.php]: Ok thanks for clarification.

**Pull request merged**